### PR TITLE
fix(ci): use kind-config.yaml in smoke test

### PR DESCRIPTION
**Pull request by amirdt22**
Perhaps this isn't there for a reason but thought it'd be easier than just asking in an issue.

### CODE REVIEW

**Comment by amirdt22** [comment]
Travis is failing, looks like an unrelated issue with `go`.

**Comment by BenTheElder** [comment]
looks like travis has a go version that is too old
sorry this took me so long.

File: kind-smoketest.sh [hunk 1]
```diff
@@ -12,6 +12,9 @@ BIN_DIR="$(mktemp -d)"
 # kind binary will be here
 KIND="${BIN_DIR}/kind"
 
+CWD="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+KIND_CONFIG="${CWD}/kind-config.yaml"
```
Comment (BenTheElder) [inline kind-smoketest.sh:16]: The kind-config should probably have a comment added that it's just a sample, we recommend most users use a single node cluster (the default) actually.

**Comment by BenTheElder** [comment]
we should use the config for consistency anyhow though

**Pull request merged**